docs/.vuepress/components/DoDont.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+<div class="do-dont">
+  <figure class="do-dont__figure do-dont__figure--do">
```

Comment:
I know design has said these will _always_ be in pairs, but it wouldn't hurt to throw a `v-if="doExample"` and `v-if="dontExample"` on these `<figure>`'s and make sure the component renders correctly with just one. I think there may be more solitary "Don't" examples than design is anticipating.

Reply:
I anticipate that this will be the case as well, I'll add some logic around this.